eclair-core/src/main/resources/reference.conf

```diff
@@ -162,6 +162,47 @@ eclair {
     }
 
     quiescence-timeout = 1 minutes // maximum time we will stay quiescent (or wait to reach quiescence) before disconnecting
+
+    channel-update {
+      // Balance thresholds at which to update the maximum HTLC amount
+      // Must be in increasing order.
+      balance-thresholds = [{
+        available-sat = 1000  // If our balance goes below this,
+        max-htlc-sat =  0     // set the maximum HTLC amount to this (or htlc-minimum-msat if it's higher).
+      },{
+        available-sat = 2000
```

Comment:
While I understand the rationale of defining every transition now that we take `max-htlc-sat = 0.5 * available-sat` (so that we have a "complete" piecewise function), defining so many buckets for small balances will make the channel updates very chatty, because almost every new HTLC will make us cross a boundary (which is quite bad because it will make it very easy to correlate channel updates with HTLCs).

I'm not sure what the best defaults would be, but I think we should have way less buckets for small amounts...

Reply:
I've reduced the number of very small buckets by using 10% for small amounts while keeping 50% for the larger amounts.